modules/assim.batch/R/pda.neff.R

```diff
@@ -0,0 +1,163 @@
+##' Autocorelation correction and efficient sample size calculation on latent process
+##'
+##' @title Calculate N_eff 
+##' @param inputs list
+##'
+##' @return inputs list, updated inputs with n_eff
+##'
+##' @author Istem Fer
+##' @export
+pda.neff.calc <- function(inputs){
+  
+  
+  for(i in seq_along(inputs)){
+    
+    n   <- inputs[[i]]$n
+    # for now we're doing autocorrelation correction on flux data only
+    # NEE, LE, FC
+    if(inputs[[i]]$variable.id %in% c(297, 298, 1000000042)){
```

Comment:
Personally, I would rather see these be a named vector or separated, informative named variables. That reduces the likelihood that they will get lost at some point. Also, it might be a good idea to make these a function argument -- that will make this script more versatile later.